priv/perf/config/test.exs

```diff
@@ -1,6 +1,25 @@
 use Mix.Config
 
+config :ethereumex,
+  url: System.get_env("ETHEREUM_RPC_URL") || "http://localhost:8545"
+
 config :load_test,
-  child_chain_url: "http://localhost:9656",
-  watcher_security_url: "http://localhost:7434",
-  watcher_info_url: "http://localhost:7534"
+  faucet_deposit_wei: trunc(:math.pow(10, 18) * 10),
+  faucet_private_key:
+    System.get_env("LOAD_TEST_FAUCET_PRIVATE_KEY") ||
+      "0xd885a307e35738f773d8c9c63c7a3f3977819274638d04aaf934a1e1158513ce",
+  child_chain_url: System.get_env("CHILD_CHAIN_URL") || "http://localhost:9656",
+  fee_wei: 1,
+  utxo_load_test_config: %{
+    utxos_to_create_per_session: 1_000,
+    transactions_per_session: 100
+  },
+  eth_vault_address: "0x4e3aeff70f022a6d4cc5947423887e7152826cf7"
```

Comment:
Should this (and the contract address below) be ENV vars like in the other config files?

Reply:
I left it that way for convenience. But instead of keeping it there I just committed a change - those env vars are set in make test now.